app/controllers/ems_common.rb

```diff
@@ -711,6 +711,7 @@ def set_form_vars
     @edit[:new][:hostname] = @ems.hostname
     @edit[:new][:emstype] = @ems.emstype
     @edit[:amazon_regions] = get_amazon_regions if @ems.kind_of?(ManageIQ::Providers::Amazon::CloudManager)
+    @edit[:google_regions] = get_google_regions if @ems.kind_of?(ManageIQ::Providers::Google::CloudManager)
```

Comment:
@Fryguy - [this comment](https://github.com/lwander/manageiq/blob/initial_google_cloud_platform_support/app/models/manageiq/providers/google/regions.rb#L15-L17) shows that there *is* an API in GCE to get the regions.